base/mpfr.jl

```diff
@@ -275,7 +275,7 @@ big(::Type{<:AbstractFloat}) = BigFloat
 function convert(::Type{Rational{BigInt}}, x::AbstractFloat)
     if isnan(x); return zero(BigInt)//zero(BigInt); end
     if isinf(x); return copysign(one(BigInt),x)//zero(BigInt); end
-    if x == 0;   return zero(BigInt) // one(BigInt); end
+    if iszero(x);   return zero(BigInt) // one(BigInt); end
```

Comment:
I have another PR that changes this, which will conflict anyways. So I prefer to see one merge first before making any additional changes.